test/playground/service_test.clj

```diff
@@ -83,52 +98,32 @@
       (is (or (.contains body  "only permitted to author and admin")
               (.contains body "Entry not in DB"))))))
 
-;;session
-
-#_(defn make-session-store
-  [reader writer deleter]
-  (reify session.store/SessionStore
-    (read-session [_ k] (reader k))
-    (write-session [_ k s] (writer k s))
-    (delete-session [_ k] (deleter k))))
-
-(defn make-service-fn
-  [session-store]
-  (::http/service-fn (http/create-servlet (assoc (service/service server/test-http-port) 
-                                                 ::http/enable-session {:store session-store}))))
 
-#_(defn service-fn
-  [sys]
-  (get-in sys [:pedestal :server ::http/service-fn]))
 
+(deftest admin-session-test
```

Comment:
this test fails.

Reply:
Found the fix, here's how I debugged it https://www.dropbox.com/s/bq2ho5vq602zojp/cider.mov?dl=0

Reply:
Please note, you will also need to remove the `session-interceptor`. Adding `session-interceptor` manually is mutually exclusive with the `::http/enable-session` option.

So add the `::http/enable-session` option in the dev map, making sure to:

* specify that the cookie store should be used
* QA that login keeps working.